ee/ucp/authorization/create-teams-with-ldap.md

```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
 ### Match Group Members (Direct Bind)
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
 synced to match the membership of the group.
 
-- **Group DN**: The distinguished name of the group from which to select users.
-- **Group Member Attribute**: The value of this group attribute corresponds to
+- **Group DN:** The distinguished name (DN) of the group from which to select users.
+- **Group Member Attribute:** The value of this group attribute corresponds to
```

Comment:
Would be great to have this information in a small table and not an overly-detailed bullet list.